core/src/main/java/org/apache/calcite/rel/core/JoinRelType.java

```diff
@@ -168,7 +172,7 @@ public CorrelateJoinType toLinq4jCorrelateJoinType() {
         "Unable to convert " + this + " to CorrelateJoinType");
   }
 
-  /** Transform this JoinRelType to Linq4j JoinType. **/
+  /** Transforms this JoinRelType to Linq4j JoinType. **/
   public JoinType toLinq4jJoinType() {
```

Comment:
If we don't include this method here we can remove (possibly in the next release) the dependencies to linq4j module from this class. I guess that there are not many clients who need to do this transformation. Our Enumerable join operators most likely need this method so we could move it somewhere in `org.apache.calcite.adapter.enumerable` package (e.g., `EnumUtils`).

Reply:
Ok for me to move this method to EnumUtils.